docs/index.md

```diff
@@ -248,6 +248,37 @@ To facilitate this workflow, the issuer can include a link to help the user down
   ]
 }
 ```
+#### on Android
+When transferring a SMART Health Card from one app to another on Android, use:
```

Comment:
```suggestion
When transferring a SMART Health Card from one app to another on Android, implement a [ContentProvider](https://developer.android.com/guide/topics/providers/content-provider-creating) to communicate the uri to a .smart-health-card, like this:
```

Reply:
@jdkizer9 , @OpangEpic - I'm unfortunately out of my depth here, gents. Is this content accurate and appropriate?

Reply:
I would word it somewhat like this:
> When transferring a SMART Health Card from one app to another on Android, send the SHC via a content provider URI pointing to the .smart-health-card.

If we'd like documentation to point them towards - the content provider overview page may be more apt:(https://developer.android.com/guide/topics/providers/content-providers). They may be using an android-provided content provider (Ex. FileProvider), and not making a custom one.

Reply:
```suggestion
When transferring a SMART Health Card from one app to another on Android, send it via a [content provider](https://developer.android.com/guide/topics/providers/content-providers) URI pointing to the .smart-health-card.
```